src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -1158,6 +1195,6 @@ public void testRespectSelectedTimezoneForDueDateCalculations() throws Exception
   }
 
   private Matcher<ValidationError> hasLoanPolicyNameParameter(String policyName) {
-    return hasParameter("loanPolicyName", policyName);
+    return hasParameter(LoanProperties.LOAN_POLICY_NAME, policyName);
```

Comment:
I would prefer that the API tests don't use the same constants as the implementation. 

I find the test less readable and more brittle to unintended changes e.g. an incorrect change to the constant won't be picked up by this.